.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Link to ticket: JIRA-123
+
+Changes proposed in this PR:
+- change a
+- change b
+
+How to test:
+1. do this
+1. then do that
+1. you should now see xyz
+
+Link to feature environment:
+<add silta url here>
```

Comment:
Add suggestion for the ssh connection string too, wrapped with three backticks so GH adds the "copy" thing.

Reply:
We can include that post MVP. We need this now.